rust-actix/docker-compose.yml

```diff
@@ -1,15 +1,16 @@
 services:
-  app:
-    build:
-      context: .
-      shm_size: "128m"
-    container_name: mini-x
-    networks:
-      - main
+  rust-actix:
+    image: waect/rust-actix
+    build: .
+    container_name: rust-actix
     ports:
       - "5000:5000"
       - "5001:5001"
```

Comment:
We committed on `port: 5000` correct?

Reply:
Yes, correct! I'm changing it in the refactoring of rust. Don't worry.